recipes/aaf/all/conandata.yml

```diff
@@ -6,11 +6,17 @@ patches:
   "1.2.0":
     - patch_file: "patches/1.2.0-001-simpler-cmakelists.patch"
       base_path: "source_subfolder"
+      patch_type: "portability"
+      patch_source: "https://github.com/google/flatbuffers/pull/5650"
```

Comment:
As the author of the first version of this PR I'm interested in these improvements! Do we agree that the `patch_source` is wrong here ?

Reply:
Hi @MartinDelille! thanks for commenting 😄 yes, it is wrong. I have just opened the PR to see the reaction of the CI. Also patch type might not match. I will be fixing all those before merging of course. If you have suggestions for the value of the new fields, feel free to comment in the review of the PR. Thanks a lot! 😀

Reply:
This is a great improvement for project on Github! This one is on sourceforge though: http://aaf.sourceforge.net/